src/lib/blockheights.js

```diff
@@ -31,10 +35,34 @@ const compareBlockHeightsCron = async () => {
   let matchingBlockHeights = 0;
 
   if (localBlockheight === "0")
+  {
+    var currentTime = new Date();
```

Comment:
Here you extending to the upper level the scope of `currentTime`, which is not needed.

https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var-to-declare-a-variable-in-jav